brooklyn-tosca-transformer/src/main/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java

```diff
@@ -212,6 +230,41 @@ private static boolean isTosca(Map<?,?> obj) {
         return rootSpec;
     }
 
+    public void decorateEntityBrooklynWithPolicies(EntitySpec<? extends Application> appSpec,
+                                                   NodeGroup group,
+                                                   GenericPolicy policy){
+        String type;
+        BrooklynClassLoadingContext loader = JavaBrooklynClassLoadingContext.create(mgmt);
+        BrooklynYamlTypeInstantiator.Factory yamlLoader =
+                new BrooklynYamlTypeInstantiator.Factory(loader, this);
+
+        type = getPolicyType(policy);
+        if(policy.getType()==null) {
+            throw new IllegalStateException("Type was nof found for policy " + policy.getName());
+        }
+
+        ImmutableMap policyDefinition = ImmutableMap.of(
+                BrooklynCampReservedKeys.BROOKLYN_POLICIES, ImmutableList.of(
+                        ImmutableMap.of(
+                                "policyType", type,
+                                BrooklynCampReservedKeys.BROOKLYN_CONFIG, getPolicyProperties(policy))));
+
```

Comment:
Next step could be change this point. Probably, the best option could be avoiding CAMP engine.